blockstore/src/main/java/brooklyn/location/blockstore/AbstractVolumeManager.java

```diff
@@ -112,6 +116,20 @@ public MountedBlockDevice mountFilesystem(AttachedBlockDevice attachedDevice, Fi
         Map<String, ?> flags = MutableMap.of("allocatePTY", true);
         int exitCode = ((SshMachineLocation)machine).execCommands(flags, "Mounting EBS volume", ImmutableList.of(
                 dontRequireTtyForSudo(),
+                ifFileExistsElse0(
```

Comment:
I wouldn't want to persons files to be backed up by this customizer.
For someusers this might be unexpected behavior and it could break application using that partition at the moment.
If the process using partition you are replacing is managed by Apache Brooklyn, somehow you should also be able to trigger some hook for stopping the Brooklyn node.
So it would be nice to have some entities supplied on this line which you can stop before umounting the old partition and start when migration is finished.
I suggest discussing that on the Apache Brooklyn mailing list if you think my suggestion is needed for you.
For the time being I am OK merging that as long as it stopped.
You could you turn off that behavior by default. with a customizer config key.
Also please replace tab characters with 4 space characters.